docs/kafka/nodejs-kafka/README.adoc

```diff
@@ -146,16 +146,16 @@ endif::[]
 
 .Procedure
 
-. In your IDE, create a new file. Save the file with the name `rhoas.env`, at the root level of the `reactive-example` directory for the cloned repository.
+. In your IDE, create a new file. Save the file with the name `.env`, at the root level of the `reactive-example` directory for the cloned repository.
 
-. In the `rhoas.env` file, add the lines shown in the example. These lines set the bootstrap server and client credentials as environment variables to be used by the Node.js application.
+. In the `.env` file, add the lines shown in the example. These lines set the bootstrap server and client credentials as environment variables to be used by the Node.js application.
 +
-.Setting environment variables in the rhoas.env file
+.Setting environment variables in the .env file
 [source,subs="+quotes"]
 ----
-KAFKA_HOST=__<bootstrap_server>__
```

Comment:
Those values will come from `rhoas cli` context. Wondering why we needed to change them?

@rkpattnaik780 did you made any changes on node.js side to support new environment variables?

Reply:
Just realized the changes hadn't been pushed. Will be raising the PR now.

The values are not obtained from the CLI, this guide is around UI. The environment variables were changed according to the current configuration file format.